Codecademy/gamut
# feat(Illustrations): Add 3 new onboarding quiz illustrations

### Overview


Add 3 new illustrations for our revamp of the onboarding quiz. 


### PR Checklist

- [x] Related to designs: https://www.figma.com/file/TjBVb65kukF5TZRqYUn4Se/onboarding?type=design&node-id=1234-28942&mode=design&t=nf4epDLvZQcEglKG-0
- [x] Related to JIRA ticket: [DISC-1218]

#### Testing Instructions


[Storybook](https://www.figma.com/file/TjBVb65kukF5TZRqYUn4Se/onboarding?type=design&node-id=1234-28942&mode=design&t=nf4epDLvZQcEglKG-0): Under Illustrations, see ~Goal~ Purpose, Interest, and Knowledge. 

[Mono PR Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2788): As a logged in user navigate to the onboarding quiz `/welcome/find-a-course`. See appropriate image and text next to each question. Details of implementation/axe will be addressed in separate mono ticket. This is just to make sure the images are being imported from Gamut ok. 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/36140)  |  |
| Portal       | [Portal PR](https://github.com/codecademy-engineering/mono/pull/2788)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-2788)   |




[DISC-1218]: https://codecademy.atlassian.net/browse/DISC-1218?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
